crates/wasmcloud-host/src/oci.rs

```diff
@@ -71,7 +75,7 @@ pub(crate) async fn fetch_provider_archive(
     img: &str,
     allow_latest: bool,
 ) -> Result<ProviderArchive> {
-    let bytes = fetch_oci_bytes(img, allow_latest).await?;
+    let bytes = fetch_oci_bytes(img, allow_latest, false).await?;
```

Comment:
You should be able to make this also a parameter of this function, you'll need to follow the call stack a few layers but it leads to `impl Handler<NatsMessage> for ControlInterface`